library/src/interactor_impl.cxx

```diff
@@ -359,6 +371,25 @@ class interactor_impl::internals
     this->Window.render();
   }
 
+  static std::string FormatInteractionBind(const InteractionBind& bind)
```

Comment:
Maybe it should be a function of `InteractionBind` struct like `std::string InteractionBind::Format() const`?